src/shortcuts/plugin.ts

```diff
@@ -245,6 +245,16 @@ const SHORTCUTS = [
     selector: '.jp-Console .jp-CellEditor',
     sequence: ['Escape']
   },
+  {
+    command: 'terminal:increase-font',
+    selector: '.jp-TerminalWidget .jp-TerminalWidget-body',
+    sequence: ['Ctrl =']
```

Comment:
How about `Accel Shift =` as a compromise so it wouldn't interfere with the browser Zoom?